### PR TITLE
Allow to have extra database for logs

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: security/db_router.py [hunk 1]
```diff
@@ -31,3 +31,33 @@ def allow_relation(self, obj1, obj2, **hints):
 
     def allow_migrate(self, db, app_label, model_name=None, **hints):
         return True
+
+
+class MultipleDBSecurityLoggerRouter:
+    """
+    A router controlling tha all that read and write database operations will be performed via mirror without atomic.
```
Comment (rubickcz) [inline security/db_router.py:38]: A router controlling **that all** read and write database operations **for apps specified in `route_app_labels`** will be performed via mirror without atomic.

File: security/locale/cs/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -8,7 +8,11 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
+<<<<<<< Updated upstream
 "POT-Creation-Date: 2020-10-05 00:01+0200\n"
+=======
+"POT-Creation-Date: 2020-10-04 23:54+0200\n"
+>>>>>>> Stashed changes
```
Comment (rubickcz) [inline security/locale/cs/LC_MESSAGES/django.po:15]: Fix this please.

File: security/db_router.py [hunk 3]
```diff
@@ -31,3 +31,33 @@ def allow_relation(self, obj1, obj2, **hints):
...
+    """
+
+    route_app_labels = [
+        'security', 'debug_toolbar_log'
+    ]
+
+    def _db_for_read_or_write(self, model):
+        if model._meta.app_label in self.route_app_labels:
+            return settings.LOG_DB_NAME
+        return None
```
Comment (rubickcz) [inline security/db_router.py:48]: Why don't you return the default DB here?

**Review by rubickcz — CHANGES_REQUESTED** [review]

File: security/db_router.py (hunk 3 above)
Comment (matllubos) [reply security/db_router.py:48]: look to the documentation. There can be more routers. Your router should decide only about your tables but other tables are left for next defined router

**Pull request merged**